src/com/futurewei/alcor/controller/db/Database.java

```diff
@@ -0,0 +1,62 @@
+package com.futurewei.alcor.controller.db;
+
+import com.futurewei.alcor.controller.db.ignite.IgniteCache;
+import com.futurewei.alcor.controller.db.redis.RedisCache;
+import com.futurewei.alcor.controller.model.PortState;
+import com.futurewei.alcor.controller.model.SubnetState;
+import com.futurewei.alcor.controller.model.VpcState;
+import org.apache.ignite.client.IgniteClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.stereotype.Component;
+
+@ComponentScan
+@Component
+public class Database {
```

Comment:
The class name "Database" is a bit generic. Use something more specific like DB

If different microservices uses different template, how are we going to leverage this class? For example, VPC manager might use vpcTemplate, Subnet Manager might use subnetTemplate etc.

Reply:
Maybe it's time to reconsider a new name.